pkg/orchestration/wiring/kubeingress/plugin.go

```diff
@@ -124,11 +125,9 @@ func buildServiceResource(deploymentName smith_v1.ResourceName, selectorLabels m
 	return serviceResource
 }
 
-// buildIngressResource constructs the Kube / KITT Ingress object
-// with a default rule, plus all aliases rules from dependants internalDNS resources
-func buildIngressResource(serviceName smith_v1.ResourceName, resource *orch_v1.StateResource, context *wiringplugin.WiringContext) (wiringplugin.WiredSmithResource, error) {
+func buildIngressResourceFromSpec(serviceName smith_v1.ResourceName, resourceName voyager.ResourceName, timeout int, context *wiringplugin.WiringContext) (*wiringplugin.WiredSmithResource, error) {
```

Comment:
`serviceName` is not the service name. It is name of the `Service` resource. Just a misleading name. Rename to something like `serviceResourceName` to make it clearer? We use `serviceName voyager.ServiceName` all over the place and this looks like a bug initially.